pkgs/tools/package-management/nix-serve/default.nix

```diff
@@ -18,7 +18,7 @@ stdenv.mkDerivation rec {
     inherit rev sha256;
   };
 
-  buildInputs = [ bzip2 perl nix ]
+  buildInputs = [ bzip2 perl nix nix.perl-bindings ]
```

Comment:
AFAIK you do not need `bzip2 perl nix` here.

Although `perl` coudl be useful for basic sanity check: `perl -c "$src/nix-serve.psgi"`

Reply:
It expects ```nix-store``` and ```bzip2``` on ```$PATH```

https://github.com/edolstra/nix-serve/blob/e4675e38ab54942e351c7686e40fabec822120b9/nix-serve.psgi#L54

Reply:
Ah, that's covered in nixos/modules/services/networking/nix-serve.nix, so you are right could be removed from here

Reply:
Runtime and build dependencies are different things.

And if you inject ${pkgs.foo}/bin/foo into output script, you do not need to add foo as a build dependency.

Reply:
[I am sure](https://github.com/ip1981/nixsap/blob/master/modules/pkgs/nix-serve/default.nix) :)

Reply:
Can you submit a PR?

Reply:
It is not worthwhile on its own.

Although, an important thing is that `$PERL5LIB` is automatically defined by `buildInputs` _and_ expanded at build time.

Also that's a kind of bug that `${perlPackages.Starman}/bin/starman` can't find its own modules, thus `Starman` is also included.